From 46d48d608225f72731cca578fdf964b3fb1a477f Mon Sep 17 00:00:00 2001
From: Hari Nagalla <hnagalla@ti.com>
Date: Wed, 12 Feb 2025 14:07:04 -0600
Subject: [PATCH 68/68] TI: arm64: dts: ti: k3-am62x-sk-common: Move ramoops
 location

Move ramoops memory carveout since at its current location
it overlaps with the rtos to rtos memory carveout. We cannot
move the rtos to rtos memory carveout due to FW hardcoding
memory regions so lets move ramoops instead.

Signed-off-by: Hari Nagalla <hnagalla@ti.com>
Signed-off-by: Judith Mendez <jm@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi b/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi
index ecfb4ce0ee5d..5eb9ff0ab019 100644
--- a/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi
@@ -39,9 +39,9 @@ reserved-memory {
 		#size-cells = <2>;
 		ranges;
 
-		ramoops@9ca00000 {
+		ramoops@9c700000 {
 			compatible = "ramoops";
-			reg = <0x00 0x9ca00000 0x00 0x00100000>;
+			reg = <0x00 0x9c700000 0x00 0x00100000>;
 			record-size = <0x8000>;
 			console-size = <0x8000>;
 			ftrace-size = <0x00>;
-- 
2.47.2

